packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -41,19 +41,19 @@ npm install @bbc/psammead-storybook-helpers --save-dev
 
 ## Usage
 
-### LANGUAGE_VARIANTS
+### TEXT_EXAMPLES
```

Comment:
What do you think about renaming this to `TEXT_VARIANTS` since we are importing from `text-variants`?

Reply:
🤔 that's a good idea